<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MandatoryGroups</class>
 <widget class="QDialog" name="MandatoryGroups">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мандатные группы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="MandatoryGroups_groupBox">
     <property name="title">
      <string>Группы</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="MandatoryGroupslistView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Информация о группе</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="MandatoryMark_label_2">
        <property name="text">
         <string>Мандатная метка:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MandatoryMark_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="RoleAtributes_groupBox">
     <property name="title">
      <string>Ролевые атрибуты</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="RoleAtributes_listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="Buttons_Layout">
     <item>
      <widget class="QPushButton" name="AddMandatoryGroup_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditMandatoryGroup_pushButton">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteMandatoryGroup_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
